docs/Configuration.md

```diff
@@ -51,6 +51,25 @@ Repeating a key name in one section is forbidden.
 
 Each section in `backend_config.ini` is documented below.
 
+## RPCAPI section
+
+Available keys: `enable_batch_jobs`, `enable_add_api_user`.
```

Comment:
To be consistent, shouldn't the first key be `enable_add_batch_jobs`. Also see comment in #850.